test/server/cards/02_SHD/upgrades/Ruthlessness.spec.ts

```diff
@@ -0,0 +1,124 @@
+describe('Ruthlessness', function () {
+    integration(function (contextRef) {
+        describe('Ruthlessness\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'battlefield-marine', upgrades: ['ruthlessness'] }, 'wampa'],
+                    },
+                    player2: {
+                        groundArena: ['atst', 'specforce-soldier', 'wilderness-fighter', 'regional-governor']
+                    }
+                });
+            });
+
+            it('should not deal 2 damage to base when not defeating a enemy unit', function () {
```

Comment:
The naming for this test seems wrong since we are testing some cases where the ability triggers successfully